### ISSUE TITLE
Get balance compared to nominal hours

### ISSUE THREAD

**Issue by KennethNielsen** [TailorDev/Watson#249]
Hallo Watson devs

I use watson to track hours at work to keep track of what I work on and how much. I'm not payed by the hour or for over time, but I still like to keep track of how much I work compared to the nominal number of hours I'm supposed to work.

For this purpose, really what I need is to know the balance i.e. how many hours have I worked in total (within a time span say month) compared to the nominal hours, which are calculated as nominal hours per day times the work days in the time span.

Does such a feature exist and if not would you be interested in it?

**Comment by jmaupetit** [comment]
This feature has already been discussed if I remember well. IMO the main painful point would be to make it optional and easily configurable. Do you have ideas about the configuration of this feature?

**Comment by KennethNielsen** [comment]
Yes. In my mind this has to be a separate sub command (say "balance"). I know that we should probably add those sparsely, but I think that it is very important here because:

 * All the rest of Watson has a related, but distinctly separate purpose, so I think it is very important that this feature has no impact on existing functionality whatsoever.

With regards to configuration, which was you real question, what I would do is just 2 new configuration keys:

1. One for the nominal number of hours per weekday

**Comment by semenmiroshnichenko** [comment]
This would be a great feature, currently I'm doing that beside of the Watson just in an Excel sheet week per week.

**Comment by KennethNielsen** [comment]
Just a littkle status update here. I'm still planning to work on this and I think I have a good plan for it, but right now it is kind of waiting for #277 since I prefer not to have many open side projects.

**Comment by KennethNielsen** [comment]
@semenmiroshnichenko BTW. Since I find Excel extremely painful I should point out that since most of Watsons commands can output JSON, you can Python script your way out of this, which in my mind is more comfortable. I created a gist for my quick and dirty script for the job, in case anyone is interested.But I would still like Watson to do it. If it is of any interest

**Comment by varac** [comment]
Related historic issue/PR:

* https://github.com/TailorDev/Watson/pull/52
* https://github.com/TailorDev/Watson/issues/397
